STUDIO/org.openl.rules.webstudio/webapp/pages/modules/administration/users/groups.xhtml

```diff
@@ -119,6 +121,30 @@
 
     </style>
 
+    <div>

```

Comment:
try to move this block to "webapp/pages/modules/administration/settings/index.xhtml"  like:
`<form>`
`<ui:include src="/pages/modules/administration/users/groups.xhtml"/>`
`<apply button>`